lomap/tests/test_new_api.py

```diff
@@ -29,10 +29,21 @@ def basic():
 
 
 def test_generate_network_smoketest(basic):
-    network = lomap.generate_lomap_network(
-        molecules=basic,
-        mappers=lomap.LomapAtomMapper(),
-        scorer=lomap.default_lomap_score,
-    )
-
-    assert isinstance(network, gufe.LigandNetwork)
+    with pytest.deprecated_call():
```

Comment:
Could you add a `match` to the warning message here? i.e. `match="molecules is deprecated"`